.github/workflows/ci.yml

```diff
@@ -27,7 +27,21 @@ jobs:
     name: test (chunk ${{ matrix.chunk }})
     strategy:
       matrix:
-        chunk: [1/8, 2/8, 3/8, 4/8, 5/8, 6/8, 7/8, 8/8]
+        chunk:
```

Comment:
When we increase the number of chunks, we also need to update https://github.com/CruGlobal/mpdx-react/blob/401ea77486bbd39a2131b8cd88c19cdb29873348/.codecov.yml#L4. And we'll want to make the new chunks required checks in Terraform.